pkg/integrations/squid_exporter/squid_exporter_test.go

```diff
@@ -85,18 +88,16 @@ func TestConfig_UnmarshalYaml(t *testing.T) {
 		require.NoError(t, yaml.UnmarshalStrict([]byte(strConfig), &c))
 
 		require.Equal(t, Config{
-			Address:  "localhost:3182",
-			Username: "",
-			Password: "",
+			Address: "localhost:3182",
 		}, c)
 	})
 
 	t.Run("all values", func(t *testing.T) {
 		strConfig := `
-address: "localhost:3182"
-username: "user"
-password: "password"
-`
+		address: "localhost:3182"
```

Comment:
I think this change is causing a test to fail:
https://github.com/grafana/agent/actions/runs/5294045551/jobs/9582928409?pr=4124#step:4:207

Reply:
Yup you're right. Will fix that!